wipp-backend-data/src/main/java/gov/nist/itl/ssd/wipp/backend/data/aimodelcard/AiModelCard.java

```diff
@@ -0,0 +1,225 @@
+    private Date version;
+
+    /*************************** FROM AIMODEL DATA ***************************/
+
+    /* AI model id corresponding to this AiModelCard object. */
+    private String aiModelId;
+
+    /* Human-readable name of this model card. */
+    private String name;
+
+    /* AI model creation date. */
+    @DateTimeFormat(iso = DateTimeFormat.ISO.DATE_TIME)
+    @JsonFormat(shape = JsonFormat.Shape.STRING, pattern = "yyyy-MM-dd'T'HH:mm:ss.SSSXXX")
+    private Date date;
+
+    /* AI framework where the model will be published. */
+    private String framework;
+
+    /***************************** FROM JOB DATA *****************************/
+
+    /* Pointer to job parameters (input collection) about data in WIPP. */
+    private Map<String, String> trainingData;
+
+    /* Pointer to job parameters (input collection) about parameters in WIPP. */
+    private Map<String, String> trainingParameters;
+
+    /*************************** FROM PLUGIN DATA ***************************/
+
+    /**
+     * The authors are the creators of the model and the primary points of
+     * contact.
+     */
+    private String author;
+
+    /* Provide basic details about the model. */
+    private String description;
+
+    /* The developers’ preferred citation for this model. */
+    private String citation;
+
+    /**
+     * Specialized resource type 'model'. E.g. Convolutional Neural Network,
+     * Supervision/Learning Method, Machine Learning Type
+     */
+    private List<String> operationType;
+
+    /* Specifies the model's architecture. E.g. Unet, Resnet50 */
+    private String architecture;
+
+    /***************** FROM TENSORBOARD LOGS (IF APPLICABLE) *****************/
+
+    /**
+     * Training data in order: total execution time, number of steps, maximum
+     * accuracy and minimum loss.
+     */
+    private Map<String, Float> training;
+
+    /**
+     * Testing data in order: total execution time, number of steps, maximum
+     * accuracy and minimum loss.
+     */
+    private Map<String, Float> testing;
+
+    /******************* FROM USER ENTRIES THROUGH THE UI *******************/
+
+    /* Name of the license being used. E.g. ['MIT', 'BSD-2-Clause'] */
+    private String license;
+
+    /******************* CONSTRUCTOR(S) *******************/
+    public AiModelCard(){}
+
+    public AiModelCard(AiModel aiModel, Job job, Plugin plugin)
+    {
+        this.owner = aiModel.getOwner();
+        this.publiclyShared = true;
+        this.version = new Date();
+
+        this.aiModelId = aiModel.getId();
+        this.name = aiModel.getName();
+        this.date = aiModel.getCreationDate();
+        this.framework = aiModel.getFramework();
+
+        this.trainingData = new HashMap<>();
+        this.trainingParameters = new HashMap<>();
+        for (var entry : job.getParameters().entrySet()) {
+            if(entry.getKey().toUpperCase().contains("DIR".toUpperCase())){
```

Comment:
Using the name of the job parameter is unreliable in this case (there is no naming convention), the parameter type should be used instead